### DISCUSSION TITLE
qnm doctor resolve duplications?

### DISCUSSION THREAD

**Discussion by sschneider-ihre-pvs** [ranyitz/qnm#127] [Q&A] [answered]
Is there any way to resolve massive duplications? Or is the only option pnpm and such?

**Comment by ranyitz** [comment, marked as answer]
👋 Hi!
qnm doctor only gives you visibility, it doesn't actually solve the duplication.

To solve duplications, you can either use your package manager ([yarn dedupe](https://yarnpkg.com/cli/dedupe/) or [npm dedupe](https://docs.npmjs.com/cli/v8/commands/npm-dedupe) [pnpm dedupe](https://pnpm.io/cli/dedupe))

In some cases, you'll need to change the required dependency version in the package.json in your projects or in one of your dependencies. See https://github.com/ranyitz/qnm/discussions/121 for additional context.

**Reply by sschneider-ihre-pvs** [reply]
I didn't even knew there is a dedupe command
